Testing/VelaptorTests/Content/Caching/SoundCacheTests.cs

```diff
@@ -0,0 +1,367 @@
+
+            this.mockShutDownReactable = new Mock<IReactable<ShutDownData>> { Name = nameof(this.mockDisposeSoundReactable) };
+
+            this.mockShutDownUnsubscriber = new Mock<IDisposable>();
+            this.mockShutDownUnsubscriber.Name = nameof(this.mockShutDownUnsubscriber);
+
+            this.mockDisposeSoundReactable = new Mock<IReactable<DisposeSoundData>>();
+            this.mockDisposeSoundReactable.Name = nameof(this.mockDisposeSoundReactable);
+        }
+
+        #region Constructor Tests
+        [Fact]
+        public void Ctor_WithNullSoundFactoryParam_ThrowsException()
+        {
+            // Arrange & Act & Assert
+            AssertExtensions.ThrowsWithMessage<ArgumentNullException>(() =>
+            {
+                var unused = new SoundCache(
+                    null,
+                    this.mockFile.Object,
+                    this.mockPath.Object,
+                    this.mockDisposeSoundReactable.Object,
+                    this.mockShutDownReactable.Object);
+            }, "The parameter must not be null. (Parameter 'soundFactory')");
+        }
+
+        [Fact]
+        public void Ctor_WithNullFileParam_ThrowsException()
+        {
+            // Arrange & Act & Assert
+            AssertExtensions.ThrowsWithMessage<ArgumentNullException>(() =>
+            {
+                var unused = new SoundCache(
+                    this.mockSoundFactory.Object,
+                    null,
+                    this.mockPath.Object,
+                    this.mockDisposeSoundReactable.Object,
+                    this.mockShutDownReactable.Object);
+            }, "The parameter must not be null. (Parameter 'file')");
+        }
+
+        [Fact]
+        public void Ctor_WithNullPathParam_ThrowsException()
+        {
+            // Arrange & Act & Assert
+            AssertExtensions.ThrowsWithMessage<ArgumentNullException>(() =>
+            {
+                var unused = new SoundCache(
+                    this.mockSoundFactory.Object,
+                    this.mockFile.Object,
+                    null,
+                    this.mockDisposeSoundReactable.Object,
+                    this.mockShutDownReactable.Object);
+            }, "The parameter must not be null. (Parameter 'path')");
+        }
+
+        [Fact]
+        public void Ctor_WithNullDisposeSoundsReactorParam_ThrowsException()
+        {
+            // Arrange & Act & Assert
+            AssertExtensions.ThrowsWithMessage<ArgumentNullException>(() =>
+            {
+                var unused = new SoundCache(
+                    this.mockSoundFactory.Object,
+                    this.mockFile.Object,
+                    this.mockPath.Object,
+                    null,
+                    this.mockShutDownReactable.Object);
+            }, "The parameter must not be null. (Parameter 'disposeSoundsReactable')");
+        }
+
+        [Fact]
+        public void Ctor_WithNullShutDownReactorParam_ThrowsException()
```

Comment:
Refactor unit test name to:

```suggestion
        public void Ctor_WithNullShutDownReactableParam_ThrowsException()
```